nx/include/net/bpf.h

```diff
@@ -0,0 +1,484 @@
+// TuxSH: removed definitions under _KERNEL ifdef blocks, modify the prototype of some functions, some other cleanup, etc.
```

Comment:
Is there any reason to pull in this thing when literally nobody is going to use it?

Reply:
It's supported by ioctl & needed to implement the drivers properly.